docs/documentation/unity/visualscripting/nodes.md

```diff
@@ -0,0 +1,143 @@
+| **Conference Alias** | Input | String | The conference [alias](xref:DolbyIO.Comms.Conference.Alias), a nickname that is used to identify the conference. |
+| **Spatial Audio Style** | Input | Spatial Audio Style | The [spatial audio style](xref:DolbyIO.Comms.SpatialAudioStyle) that defines how the spatial location should be communicated between the SDK and the Dolby.io platform. By default, the parameter is set to `shared` indicating the client application just reports its own position within the virtual world.|
+
+---
+### Demo
+
+The Demo conference node allows connecting to a conference that has several bots injecting audio. This makes it easier for testing the connection during the prototyping phase.
+<div style="text-align:left">
+    <img style="padding:25px 0" src="~/images/nodes/demo.png" width="200px">
+</div>
+
+| Name  | Direction | Type | Description  |
+|---|:---|:---|:---|
+| **Scale, Forward, Up, Right** | Input | Vector3| [Vectors](xref:DolbyIO.Comms.Services.ConferenceService.SetSpatialEnvironmentAsync(System.Numerics.Vector3,System.Numerics.Vector3,System.Numerics.Vector3,System.Numerics.Vector3)) that define the 3D environment that you are working with. The default values are based on the Unity Coordinates System. In most cases, you should only modify the scale.|
+| **Spatial Audio** | Input | Boolean | Whether you want to enable spatial audio for the demo| 
+
+---
+### Mute Participant
+
+Allows muting a participant. This node works for both local participant (e.g., mute myself) and remote participant (e.g., mute others for me). 
+<div style="text-align:left">
+    <img style="padding:25px 0" src="~/images/nodes/mute.png" width="250px">
+</div>
+
+| Name  | Direction | Type | Description  |
+|---|:---|:---|:---|
+| **ParticipantId** | Input | String| The ID of the participant who should be muted. If the ID is not provided, the node mutes the local participant|
+| **Muted** | Input | Boolean | The required mute state.| 
+
+---
+### Local Player Position
+
+Allows setting the position of the local player.
+<div style="text-align:left">
+    <img style="padding:25px 0" src="~/images/nodes/local-position.png" width="250px">
+</div>
+
+| Name  | Direction | Type | Description  |
+|---|:---|:---|:---|
+| **Position** | Input | Vector3| The [position](xref:DolbyIO.Comms.Services.ConferenceService.SetSpatialPositionAsync(System.String,System.Numerics.Vector3)) of the player.|
+| **Direction** | Input | Vector3 | The [direction](xref:DolbyIO.Comms.Services.ConferenceService.SetSpatialDirectionAsync(System.Numerics.Vector3)) that the player should be facing.| 
+
+---
+### Remote Player Position
+
+Set the remote participant's spatial audio position for the local participant. This is only applicable when the spatial audio style was set as `individual` in the Spatial Conference node where each client needs to inform the server how they would like the platform to render the remote participant's audio for them.  
```

Comment:
I think we need to add: 'or when the demo is run without spatial audio'

Reply:
I see, so `Spatial Audio` in the Demo Conference node maybe a bit misleading then, if I don't run with spatial audio for demo, why would I be able to set remote participant position?

Reply:
My mistake, your are right, no spatial audio in the demo means no spatial audio..., not individual mode.